test/server-test.js

```diff
@@ -87,6 +87,31 @@ describe('Server', function(){
             })
       })
     })
+
+    describe("POST /foods", function(){
```

Comment:
here's our POST test. It's technically passing, but we're asserting the response from the server is hardcoded to the happy path.

We're working on adding the sad path, but this is a struggle.